src/opencv.d.ts

```diff
@@ -1540,7 +1540,7 @@ declare module opencv {
         convexityDefects(contour: Mat, convexHull: Mat, convexityDefects: Mat): void;
         findContours(
             image: Mat | MatVector,
-            contours: Mat | MatVector,
+            contours: MatVector,
```

Comment:
Am i missing something? The TS definition here will accept either a Mat or MatVector

Reply:
I thought too, and tested it to see if it worked with "Mat", but it didn't.
Then I noticed that in the documentation there is an array of array of points, and for that it needs to be "MatVector".

contours | Detected contours. Each contour is stored as a vector of points (e.g. std::vector<std::vector<cv::Point> >)

![image](https://user-images.githubusercontent.com/1814990/165533593-55cac5ef-301f-425f-9f0c-4d24bd30b395.png)

Reply:
Yup, tested and confirmed. I'll update the npm package with this PR

Reply:
> Yup, tested and confirmed. I'll update the npm package with this PR. Thanks for checking this out.

Yes I will probably test all the functions.
Because I'm creating a visual and online tool based on your project. :)

https://online.opencvflow.org/